beta/examples/tensorflow/common/prepare_checkpoint.py

```diff
@@ -0,0 +1,106 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import sys
+
+import tensorflow as tf
+
+from beta.nncf import create_compressed_model
+from beta.examples.tensorflow.common.logger import logger
+from beta.examples.tensorflow.common.sample_config import create_sample_config
+from beta.examples.tensorflow.common.argparser import get_common_argument_parser
+from beta.examples.tensorflow.common.utils import configure_paths
+from beta.examples.tensorflow.object_detection.models.model_selector import get_predefined_config
+from beta.examples.tensorflow.object_detection.models.model_selector import get_model_builder
+
+
+def get_argument_parser():
+    parser = get_common_argument_parser(precision=False,
+                                        save_checkpoint_freq=False)
+
+    parser.add_argument(

```

Comment:
It looks like, the mode command line attribute is redundant here.

Reply:
That is true, actually, the whole `get_argument_parser` function here is redundant.